dev/docs/styles.html

```diff
@@ -14,7 +15,17 @@
   }
 
   header {
-    background-color: #01579b;
+    background-color: white;
+    color: #424242;
+  }
+
+  nav.navbar {
+    min-height: 57px;
+    padding: 6px 0;
+  }
+
+  nav.nav.navbar-fixed-top {
```

Comment:
I think the selector here needs to be `header.header-fixed nav.navbar-fixed-top` - the shadow under the header doesn't quite match the new one on flutter.io.